frontend/src/libs/components/link/link.tsx

```diff
@@ -6,7 +6,7 @@ import { type ValueOf } from '#libs/types/types.js';
 import styles from './styles.module.scss';
 
 type Properties = {
-  to: ValueOf<typeof AppRoute>;
+  to: ValueOf<typeof AppRoute> | `${ValueOf<typeof AppRoute>}?query=${string}`;
```

Comment:
```suggestion
  to: ValueOf<typeof AppRoute>;
```

Reply:
@Liza-Veis without it i can`t pass url with query params

Reply:
done